Portland CFP WIP (#1641)
writethedocs/www: docs/conf/portland/2022/cfp.rst

:template: {{year}}/generic.html

Call for Proposals
==================

> Hello hello, fellow documentarians! It's that time of year again: We’re very excited to announce that we are now accepting talk proposals for our next {{ city }} conference, coming up on {{date.main}}. We're running a `hybrid conference <https://www.writethedocs.org/conf/portland/2022/news/welcome/>`__ this year, so you can speak or attend from anywhere, but folks will also be there in person in {{ city }}. We will use regular {{ city }} conference times for the scheduling.

Reviewer: > We will use regular {{ city }} conference times for the scheduling.

Not sure what this means?

Author: I think it's just about confirming that the "Portland" conference is on the Portland timezone, even when hybrid or virtual.

> Hello hello, fellow documentarians! It's that time of year again: We’re very excited to announce that we are now accepting talk proposals for our next {{ city }} conference, coming up on {{date.main}}. We're running a `hybrid conference <https://www.writethedocs.org/conf/portland/2022/news/welcome/>`__ this year, so you can speak or attend from anywhere, but folks will also be there in person in {{ city }}. We will use regular {{ city }} conference times for the scheduling.

Author: ```suggestion
Hello hello, fellow documentarians! It's that time of year again: We’re very excited to announce that we are now accepting talk proposals for our next {{ city }} conference, coming up on {{date.main}}. We're running a `hybrid conference <https://www.writethedocs.org/conf/portland/2022/news/welcome/>`__ this year, so you can speak or attend from anywhere, but folks will also be there in person in {{ city }}. Schedules will be in {{ tz }}.
```

>     For speakers this means you'll be writing and recording your talk *before the event*, the same as for our virtual conference. We'll have a videographer around to help with that if you want it. we're working on the exact details of a moderated remote live Q&A after your talk.

Reviewer: ```suggestion
    For speakers this means you'll be writing and recording your talk *before the event*, the same as for our virtual conference. We'll have a videographer around to help with that if you want it. We'll also have a live, moderated Q&A after your talk with our friendly MC. 
```

Author: I've used `emcee` elsewhere, otherwise :+1: +1

>     For speakers this means you'll be writing and recording your talk *before the event*, the same as for our virtual conference. We'll have a videographer around to help with that if you want it. we're working on the exact details of a moderated remote live Q&A after your talk.

Reviewer 2: ```suggestion
For speakers, this means you'll be writing and recording your talk *before the event*, the same as for our virtual conference. Our video producer will be available to help with recording if you want it. We're working on the exact details of a moderated remote live Q&A after your talk.
```

Reviewer 2: I removed the spaces before this paragraph but then I looked at the preview and saw that it ended up being a different font and alignment, so feel free to edit my suggestion and add the spaces back if you want :)

Every year, Write the Docs invites people from all across our community to come up on stage to share their insights and experience. Whether you've worked on documentation for decades or you've just started this year, we want to hear from you!
Read on to learn more about the goals of the conference and what we look for in talk proposals.

{% if flagcfp %}
In the meantime, mark your calendars:

> **The deadline for submitting proposals is Midnight {{tz}} on {{cfp.ends}}.**

Reviewer: ```suggestion
**The deadline for submitting proposals is midnight {{tz}} on {{cfp.ends}}.**
```

> **The deadline for submitting proposals is midnight {{tz}} on {{cfp.ends}}.**

Reviewer: ```suggestion
**The deadline for submitting proposals is 11:59PM {{tz}} on {{cfp.ends}}.**
```

To match

We'll let you know if your proposal has been accepted by the end of {{cfp.notification}}.
{% else %}
We will announce the CFP soon.
{% endif %}

.. contents::
    :local:
    :depth: 1
    :backlinks: none

Ready to submit your talk?

> You can do that here:

Reviewer 2: Should we soft-launch pretalx to test these buttons?

.. raw:: html

    <div class="announcement" style="background-color:white;">
        <div class="uk-container">
        <a style="border-bottom: none; font-size: .875rem;" class="uk-button uk-button-announcement uk-text-center" href="{{ cfp.url }}">Submit your proposal</a>
        </div>
    </div>

Conference goals
----------------

Write the Docs conferences are our chance to get together to explore and celebrate the craft of documentation in a positive, inclusive environment.

> Between the talks on stage, the discussions in the unconference, the collaboration on the writing day and other conversations, 100% of the content for our conferences comes from our community! Whether your job title is writer, developer, product manager, support advocate, librarian, or one of a hundred others, your perspective on what makes good docs is what this conference is all about.

Reviewer 2: ```suggestion
Between the talks on the stage, the discussions in the unconference, the collaboration on the writing day and other conversations, 100% of the content for our conferences comes from our community! Whether your job title is writer, developer, product manager, support advocate, librarian, or one of a hundred others, your perspective on what makes good docs is what this conference is all about.
```

If you need a hand preparing or honing your talk proposal, there are lots of good places to start:

* **Community mentorship** – We have an ever-growing pool of previous Write the Docs speakers, many of whom are happy to be a second pair of eyes on talk proposals. If you're interested in working with a past speaker, let us know at {{ shortcode }}@writethedocs.org!
* **Meetup brainstorming** – For some in-person workshopping, check in on your `local meetup group <https://www.writethedocs.org/meetups/>`_ and see if they have a talk brainstorming session on their calendar. If they don't... ask if they're planning one!
* **Slack hivemind** – You can also hit up the hivemind directly on the Write the Docs Slack, any time of day! (If you're not registered yet, you can at `http://slack.writethedocs.org/ <http://slack.writethedocs.org/>`_.)
* **Twitter hivemind** – If Twitter is more your speed, `#writethedocs <https://twitter.com/hashtag/writethedocs>`__ will get you there.

> Because the conference is virtual this year, there is no need for travel, and we're hoping this will make our CFP accessible to more documentarians. We will also provide resources and support for recording your talk.

Reviewer: ```suggestion
Because the conference is hybrid this year, there is no requirement for travel, and we're hoping this will make our CFP accessible to more documentarians. We will also provide resources and support for recording your talk.
```

> Because the conference is virtual this year, there is no need for travel, and we're hoping this will make our CFP accessible to more documentarians. We will also provide resources and support for recording your talk.

Reviewer 2: ```suggestion
Because the talks are pre-recorded, you can submit your proposal even if you can't attend in person! We will use our virtual conference platform to allow folks to attend from home globally, and we hope that this will make our CFP accessible to more documentarians. We will also provide resources and support for recording your talk.
```

Selection process
------------------

We have a small panel of proposal reviewers, and make sure to have a similar diversity in the panel as we're aiming for in our speakers.
We rate talks out of five, and then discuss the top rated proposals.

We actively balance for diversity in as many ways as we can, which means that we do not review talks anonymously. Maybe one day the industry will be in a place where can do that, but we're certainly not there yet.

Presentation format
-------------------

> Presentations will be scheduled in 30-minute blocks. As the conference is virtual, all talks will be pre-recorded. We will offer resources and support for making your talk recording. After your talk, there will be a live Q&A session. You can opt out of the Q&A if you do not feel comfortable, but please let us know well in advance.

Reviewer: ```suggestion
Presentations will be scheduled in 30-minute blocks. As the conference is hybrid, all talks will be pre-recorded. We will offer resources and support for making your talk recording. After your talk, there will be a live Q&A session. You can opt out of the Q&A if you do not feel comfortable, but please let us know well in advance.
```

> Presentations will be scheduled in 30-minute blocks. As the conference is virtual, all talks will be pre-recorded. We will offer resources and support for making your talk recording. After your talk, there will be a live Q&A session. You can opt out of the Q&A if you do not feel comfortable, but please let us know well in advance.

Reviewer 2: ```suggestion
Presentations will be scheduled in 30-minute blocks, so your recording needs to fit within that time slot. After your talk, there will be a live Q&A session. You can opt out of the Q&A if you do not feel comfortable, but please let us know well in advance.
```

Speaker benefits & logistics
----------------------------

> If you are selected to speak at Write the Docs, we will waive your attendance fee. As the conference is virtual, there are no travel costs.

Reviewer: ```suggestion
If you are selected to speak at Write the Docs, we will waive your attendance fee.
```

> If you are selected to speak at Write the Docs, we will waive your attendance fee. As the conference is virtual, there are no travel costs.

Reviewer 2: ```suggestion
If you are selected to speak at Write the Docs, we will waive your ticket cost. If you attend virtually, there are no travel costs.```

> If speaking incurs any costs that are difficult for you to cover, please `let us know <mailto:{{email}}>`_ and we'll do our best to help out.

Reviewer 2: ```suggestion
If you want to attend in person or if speaking incurs any costs that are difficult for you to cover, please `let us know <mailto:{{email}}>`_ and we'll do our best to help out.
```

If you already have a ticket, we will of course refund it - just drop us an email at `{{email}} <mailto:{{email}}>`_.

{% if flagcfp %}**You’ll hear from us with our proposal decisions by the end of {{cfp.notification}}.**{% endif %}

> All talks will be shown prerecorded, and we'll be asking for a **completed video from you by {{cfp.video_by}}**. We have a host of options to support you in making this happen, including the possibility of a live recording call with our videographer. During the conference we'll ask you to participate in a moderated Q&A video session after your talk recording is shown.

Reviewer 2: ```suggestion
All talks will be shown pre-recorded, and we'll be asking for a **completed video from you by {{cfp.video_by}}**. We have a host of options to support you in making this happen, including the possibility of a live recording call with our videographer. During the conference we'll ask you to participate in a moderated Q&A video session after your talk recording is shown.
```

> Note that all Speakers must read, understand, and agree to our :doc:`/code-of-conduct`. All talks and slides will need to follow our Code of Conduct. If you are unsure about any aspect of this, please ask us for clarification.

Reviewer 2: ```suggestion
Note that all speakers must read, understand, and agree to our :doc:`/code-of-conduct`. All talks and slides will need to follow our Code of Conduct. If you are unsure about any aspect of this, please ask us for clarification.
```
